doc-content/enterprise-only/project-data/new-project-proc.adoc

```diff
@@ -0,0 +1,14 @@
+[id='new_project']
+= Creating a project
+
+A project can contain data models, business processes, business rules, and forms that are specific to your project's requirements. When you create a project in {CENTRAL} it is added to the Git repository connected to {CENTRAL}.
```

Comment:
At present, I don't believe you can have a separate "Git repository connected to central" (except by way of git hooks). The project is created in the Git repository built into Central, I think.

Reply:
Revised to clarify.